.travis.yml

```diff
@@ -46,6 +46,10 @@ jobs:
       script: npm run docs
       perl: '5.26'
 
+    # Commit Messages
+    -
+      script: commitlint-travis
```

Comment:
Seems like this actually does everything the `lint-commits.sh` does. Can we just get rid of that file entirely? 🤔

Reply:
Was in the process of it... Just wanted to check if it worked first, hence the intermediate commit. Check now

Reply:
Ah, I see. 😅